substrate-archive/src/database/models.rs

```diff
@@ -219,6 +258,17 @@ impl PersistentConfig {
 		let version = semver::Version::parse(version)?;
 		Ok((version.major.try_into()?, version.minor.try_into()?, version.patch.try_into()?))
 	}
+
+	pub(crate) fn chain(&self) -> Chain {
+		match self.chain.to_ascii_lowercase().as_str() {
+			"kusama" => Chain::Kusama,
+			"polkadot" => Chain::Polkadot,
+			"westend" => Chain::Westend,
+			"centrifuge" => Chain::Centrifuge,
```

Comment:
I haven't heard of centrifuge before!

Reply:
aka Altair on Kusama. :)

Reply:
Yup, Altair

they're defined here because they also exist directly in PolkadotJS defs too

Reply:
Aah ok, that's handy to know! Thanks!